scripts/create_x509_chain_pipeline.py

```diff
@@ -0,0 +1,434 @@
+import os
+import re
+import base64
+import logging
+import shutil
+import subprocess
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Do we want to have logs on?

Reply:
not necessarily but after i faced the pipeline issues I thought I will change the print statements to logger so that in case something is wrong we may be bale to see it..not sure if this is a good approach or not. the logger was introduced just due to that..The print statements do not work always and gets swallowed many times.

---
In reply to: [314905023](https://github.com/Azure/azure-iot-sdk-python-preview/pull/167#discussion_r314905023) [](ancestors = 314905023)

Reply:
I will keep it for now as I am still writing tests

---
In reply to: [314908139](https://github.com/Azure/azure-iot-sdk-python-preview/pull/167#discussion_r314908139) [](ancestors = 314908139,314905023)